src/services/product.services.new.test.ts

```diff
@@ -0,0 +1,74 @@
+import { faker } from '@faker-js/faker';
+import { db } from '../../prisma/db';
+import { ApiError } from '../config/apiError';
+import { errors } from '../config/errors';
+import { ProductSerializer } from '../serializers/product-serializer';
+import { productRandomRaw } from '../test/test-constants';
```

Comment:
what do you think if we change the name of this file to test-generators or test-data-generators? Or sth like that

Reply:
Yes, I like `test-data-generators` name. I will change it